portal/eproms/views.py

```diff
@@ -298,6 +298,9 @@ def contact():
         for org in Organization.query.filter(Organization.email.isnot(None)):
             if '@' in org.email:
                 recipient_types.append((org.name, org.email))
+        config_recipients = current_app.config.get('CONTACT_RECIPIENTS', None)
```

Comment:
`get()` should return `None` by default (if a key is missing)
```suggestion
        config_recipients = current_app.config.get('CONTACT_RECIPIENTS')
```